code/datums/wires/_wires.dm

```diff
@@ -52,7 +52,7 @@
 
 /datum/wires/Destroy()
 	holder = null
-	assemblies = list()
+	QDEL_LIST_ASSOC_VAL(assemblies)
```

Comment:
I'm not sure if this is right, since it qdels all assemblies attached to wires. Those assemblies are also stored in the contents of whatever the wires are part of, so maybe it should be the responsibility of the holder to clean that up?

Reply:
```suggestion
	assemblies.Cut()
```